homeassistant/components/zwave_js/__init__.py

```diff
@@ -262,7 +276,8 @@ def async_on_value_notification(notification: ValueNotification) -> None:
                 ATTR_NODE_ID: notification.node.node_id,
                 ATTR_HOME_ID: client.driver.controller.home_id,
                 ATTR_ENDPOINT: notification.endpoint,
-                ATTR_DEVICE_ID: device.id,  # type: ignore
+                ATTR_DEVICE_ID: device.id,
+                ATTR_ENTITY_ID: entity_ids_for_device_id(ent_reg, device.id),
```

Comment:
What's the use case knowing the entity ids? I don't like this change since the entities are not the source of this event.

Reply:
from observing conversations in zwave, users often have a primary `entity ID` for a device that sends something a `notification` or `value notification` event. For `notification`, it is typically a lock, and for value notifications, it is often a switch of some sort. At one point I had proposed an idea of having a primary entity for each device, but that would be difficult to get right, so the idea here was to pass all entities for a given device to make it easier for users to create match conditions for these events in automations.

All that being said, they could use the `device_entities` function to achieve the same thing, so maybe it's not worth adding redundant info